src/neuroconv/tools/nwb_helpers/_metadata_and_file_helpers.py

```diff
@@ -370,3 +375,51 @@ def configure_and_write_nwbfile(
 
     with IO(output_filepath, mode="w") as io:
         io.write(nwbfile)
+
+
+def configure_and_export_nwbfile(
+    nwbfile: NWBFile,
+    export_nwbfile_path: Path,
+    backend_configuration: BackendConfiguration,
+) -> None:
+    configure_backend(nwbfile=nwbfile, backend_configuration=backend_configuration)
+
+    IO = BACKEND_NWB_IO[backend_configuration.backend]
+    nwbfile.set_modified()
+    with IO(export_nwbfile_path, mode="w") as io:
+        io.export(nwbfile=nwbfile, src_io=nwbfile.read_io, write_args=dict(link_data=False))
```

Comment:
Can you explain what `link_data=False` does here?

Reply:
`link_data=False` ensures that the cache of builders is cleared for each neurodata object so that they can be overwritten with the new backend information.

Reply:
It seems like the code is almost exactly the same as `configure_and_write_nwbfile`. Would it be better to include this in that function by including an `export` flag?